frontend/catalyst/api_extensions/control_flow.py

```diff
@@ -239,7 +240,31 @@ def conditional_fn():
         ...     return cond_fn()
         >>> f(1.6)
         Array(2.56, dtype=float64)
+
+    .. note::
+
+       ``catalyst.cond`` is not supported in program capture mode. If ``qml.capture`` is enabled,
+        please use ``qml.cond`` instead.
+
+        .. code-block:: python
+
+            # This will raise an error with capture mode
+            @qjit
+            def func(x):
+                @catalyst.cond(x > 1.0)
+                def cond_fn():
+                    return x ** 2
+                return cond_fn()
+
+            # Use this instead for capture mode compatibility
+            @qml.qnode(device)
```

Comment:
Why not `qjit`? Otherwise the programs are two entirely different things.
```suggestion
            @qjit
```